### PR TITLE
Add parameter badges

### PR DESCRIPTION
**Pull request by Falke-Design**
Makes the selected parameters more visible

grafik

### CODE REVIEW

File: src/components/Station.vue [hunk 1]
```diff
@@ -97,4 +111,16 @@ const { isFetching, error, data } = useFetch(
...
+  return parameters.value.indexOf(parameter.name) > -1;
+}
 </script>
+<style>
+.no-border-radius {
+  border-radius: unset;
+}
+.no-padding {
+  padding: 0;
+}
```
Comment (simon04) [inline src/components/Station.vue]: There's `p-0` in Bootstrap :) – https://getbootstrap.com/docs/5.2/utilities/spacing/#margin-and-padding

File: src/components/Station.vue [hunk 2]
```diff
@@ -97,4 +111,16 @@ const { isFetching, error, data } = useFetch(
...
 ).json<StationGeoJSONSerializer>();
+
+function filterSelectedParams(parameter: ParameterMetadataModel) {
+  return parameters.value.indexOf(parameter.name) > -1;
+}
 </script>
+<style>
+.no-border-radius {
+  border-radius: unset;
+}
```
Comment (simon04) [inline src/components/Station.vue]: There's `rounded-0` in Bootstrap: https://getbootstrap.com/docs/5.2/utilities/borders/#sizes

File: src/components/Station.vue [hunk 3]
```diff
@@ -12,11 +12,25 @@
...
+        v-model="parameters"
+      >
+        <option v-for="p in props.parameters" :value="p.name">
+          {{ p.name }} [{{ p.unit }}] ({{ p.long_name }})
+        </option>
+      </select>
+      <div>
+        <span
+          v-for="p in props.parameters"
+          v-show="filterSelectedParams(p)"
```
Comment (simon04) [inline src/components/Station.vue]:
```suggestion
          v-show="parameters.includes(p.name)"
```

- Inline this function as it is so short?
- Array.indexOf can be Array.includes.

**Review by simon04 — COMMENTED** [review]
Cool, thanks!

I'm not super happy with the `<select multiple>` usability (as you identified too). But it was the simplest solution to get started.

An alternative spot for the badges could be next to the "Parameters" label: 
image

File: src/components/Station.vue (hunk 1 above)
Comment (Falke-Design) [reply src/components/Station.vue]: Thanks

File: src/components/Station.vue (hunk 3 above)
Comment (Falke-Design) [reply src/components/Station.vue]: Didn't know that it can be inline. Makes totally sense

**Comment by Falke-Design** [comment]
> I'm not super happy with the `<select multiple>` usability

I also thought about other solutions like a Select-Dialog but they need always more clicks to choose. 

> An alternative spot for the badges could be next to the "Parameters" label:

If you select all parameters, they are too much:
grafik

**Review by simon04 — APPROVED** [review]
Cool! I'll merge this as a building block for #11.

**Pull request merged**